k8s-install/scripts/install_cni_test.go

```diff
@@ -42,6 +42,7 @@ func runCniContainer(extraArgs ...string) error {
 		"-v", cwd + "/tmp/bin:/host/opt/cni/bin",
 		"-v", cwd + "/tmp/net.d:/host/etc/cni/net.d",
 		"-v", cwd + "/tmp/serviceaccount:/var/run/secrets/kubernetes.io/serviceaccount",
+		"-v", cwd + "/tmp/etcd-cert:/calico-secrets/etcd-cert",
```

Comment:
Looks like we now require the `calico-secrets/etcd-cert` file to exist? I think we need to function even when that file is not present.